client/src/styles/buttonStyling.css

```diff
@@ -0,0 +1,16 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+body {
+  font-family: "Montserrat", sans-serif;
+}
+
+.button-style .ui.primary.button {
```

Comment:
You need to create a component called for example `CustomButton` then create style.css file for this on the same `CustomButton` component directory and import it into `CustomButton` file on the top like this `import './style.css'`